File: backend spring/src/main/java/com/bloodbank/dto/response/DonationAppointmentDto.java
package com.bloodbank.dto.response;

import lombok.Getter;
import lombok.Setter;
import java.time.LocalDate;
import java.time.LocalDateTime;

@Getter
@Setter
public class DonationAppointmentDto {
    private Long appointmentId;
    private DonorSimpleDto donor;
    private LocalDate appointmentDate;
    private String status;
    private LocalDateTime createdAt;
}

File: BackendDotNet/DTOs/Response/DonationAppointmentDto.cs
namespace BackendDotNet.DTOs.Response
{
    public class DonationAppointmentDto
    {
        public long AppointmentId { get; set; }
        public DonorSimpleDto Donor { get; set; } = new();
        public DateTime AppointmentDate { get; set; }
        public string Status { get; set; } = string.Empty;
        public DateTime CreatedAt { get; set; }
    }
}
